### PR TITLE
Explosive Hug Chemical (AKA antisocial toxin)

### PR DESCRIPTION
**Pull request by silicons**
Saw this on VGstation
I think it's hilarious
Adds an adminspawn chemical called barocardone that explodes when the mob having it gets hugged (atleast humanoids anyways. honk.) 
To spawn this just type in "antisocial" in spawn box.
Blows up the same independant of amount inside the mob. Mostly so you can't suicide maxcap by going around and hugging your target :^)
:cl:
rscadd: There are unconfirmed rumors of syndicate scientists creating a volatile chemical that will spontaneously explode in the presence of endorphins. What kind of monster would do this?
/:cl:

### CODE REVIEW

**Comment by Shapsy** [comment]
I want clappers to be real

File: code/modules/reagents/chemistry/recipes/pyrotechnics.dm [hunk 1]
```diff
@@ -409,3 +409,13 @@
 /datum/chemical_reaction/pyrosium/on_reaction(datum/reagents/holder, created_volume)
 	holder.chem_temp = 20 // also cools the fuck down
 	return
+
+/datum/chemical_reaction/hugplosion
+	name = "hugexplosion"
+	id = "hugexplosion"
+	result = "null"
+	required_reagents = list(barocardone = 1, endorphins = 1)
```
Comment (Xhuis) [inline code/modules/reagents/chemistry/recipes/pyrotechnics.dm]: Forgot quotes here

**Comment by kevinz000** [comment]
btw barocardone is a bogged spelling of baricaridine or whatever it is in vgstation

**Comment by Fox-McCloud** [comment]
Don't really understand the point of this being added.

**Comment by kevinz000** [comment]
the point is to shoot peopel with it and see who hugs them first

**Comment by KorPhaeron** [comment]
Adding a useless chem to people every time they get hugged for the sake of an adminspawn only joke seems a bit slowflake

**Comment by coiax** [comment]
Eh, I dunno. We can make people overdose and/or addicted to endorphins. We could tweak the metabolisation rates and offer very very small chances of minor stun reduction and/or stamina healing. Possibly a "You feel happy." small random message. Gotta increase the love on the station, ya know

**Comment by optimumtact** [comment]
What kor meant to say was no

**Pull request closed without merging**